src/controls/open-webui.js

```diff
@@ -0,0 +1,11 @@
+import {shell} from 'electron'
+import {apiAddrToUrl} from './utils'
+import {logger} from '../config'
+
+export default function openWebUI (ipfs, cb) {
+  ipfs().config.get('Addresses.API')
+    .then((res) => {
+      shell.openExternal(apiAddrToUrl(res))
+    })
```

Comment:
nitpick, you can reduce this to just one line: `.then((res) => shell.openExternal(apiAddrToUrl(res)))`